### PR TITLE
Update preprocessor API

### PR DESCRIPTION
**Pull request by vincentschen**
Fix tutorials in conjunction with api change: https://github.com/HazyResearch/snorkel/pull/1351

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #### ESSENTIAL LIBRARIES
 
-git+https://github.com/HazyResearch/snorkel.git@redux
+git+https://github.com/HazyResearch/snorkel.git@move_preprocessor
```
Comment (vincentschen) [inline requirements.txt]: note: will follow this up with a PR to change back to `redux`— this will avoid any breaking builds in the transition period

**Comment by vincentschen** [comment]
+ @henryre because you reviewed the relevant changes in `snorkel`

File: requirements.txt (hunk 1 above)
Comment (henryre) [reply requirements.txt]: why don't we just land redux first?

Comment (vincentschen) [reply requirements.txt]: sure, I can do that. was thinking tests/reviews might take more time, but I'll just merge them at the same time.

Comment (vincentschen) [reply requirements.txt]: also, tests will break on `tox` if this is set to redux before the merge

**Pull request merged**